plugins/jupyter-integration/src/main/java/org/polypheny/db/jupyter/MimetypeFromExtension.java

```diff
@@ -0,0 +1,232 @@
+        extensions.put( "doc", "application/msword" );
+        extensions.put( "drw", "application/drafting" );
+        extensions.put( "dvi", "application/x-dvi" );
+        extensions.put( "dwg", "application/acad" );
+        extensions.put( "dxf", "application/dxf" );
+        extensions.put( "dxr", "application/x-director" );
+        extensions.put( "eps", "application/postscript" );
+        extensions.put( "etx", "text/x-setext" );
+        extensions.put( "exe", "application/octet-stream" );
+        extensions.put( "ez", "application/andrew-inset" );
+        extensions.put( "f", "text/plain" );
+        extensions.put( "f90", "text/plain" );
+        extensions.put( "fli", "video/fli" );
+        extensions.put( "flv", "video/flv" );
+        extensions.put( "gif", "image/gif" );
+        extensions.put( "gl", "video/gl" );
+        extensions.put( "gtar", "application/x-gtar" );
+        extensions.put( "gz", "application/x-gzip" );
+        extensions.put( "hdf", "application/x-hdf" );
+        extensions.put( "hh", "text/plain" );
+        extensions.put( "hqx", "application/mac-binhex40" );
+        extensions.put( "h", "text/plain" );
+        extensions.put( "htm", "text/html" );
+        extensions.put( "html", "text/html" );
+        extensions.put( "ice", "x-conference/x-cooltalk" );
+        extensions.put( "ief", "image/ief" );
+        extensions.put( "iges", "model/iges" );
+        extensions.put( "igs", "model/iges" );
+        extensions.put( "ips", "application/x-ipscript" );
+        extensions.put( "ipx", "application/x-ipix" );
+        extensions.put( "jad", "text/vnd.sun.j2me.app-descriptor" );
+        extensions.put( "jar", "application/java-archive" );
+        extensions.put( "jpeg", "image/jpeg" );
+        extensions.put( "jpe", "image/jpeg" );
+        extensions.put( "jpg", "image/jpeg" );
+        extensions.put( "js", "text/x-javascript" );
+        extensions.put( "json", "application/json" );
+        extensions.put( "kar", "audio/midi" );
+        extensions.put( "latex", "application/x-latex" );
+        extensions.put( "lha", "application/octet-stream" );
+        extensions.put( "lsp", "application/x-lisp" );
+        extensions.put( "lzh", "application/octet-stream" );
+        extensions.put( "m", "text/plain" );
+        extensions.put( "m3u", "audio/x-mpegurl" );
+        extensions.put( "man", "application/x-troff-man" );
+        extensions.put( "me", "application/x-troff-me" );
+        extensions.put( "mesh", "model/mesh" );
+        extensions.put( "mid", "audio/midi" );
+        extensions.put( "midi", "audio/midi" );
+        extensions.put( "mif", "application/x-mif" );
+        extensions.put( "mime", "www/mime" );
+        extensions.put( "mjs", "text/javascript" );
+        extensions.put( "movie", "video/x-sgi-movie" );
+        extensions.put( "mov", "video/quicktime" );
+        extensions.put( "mp2", "audio/mpeg" );
+        extensions.put( "mp2", "video/mpeg" );
```

Comment:
@gartens Is this duplicate intentional. According to the reference linked above, it should be removed.